src/test/kotlin/no/nav/syfo/sykmelding/SykmeldingServiceTest.kt

```diff
@@ -40,343 +42,423 @@ import org.amshove.kluent.shouldBeEqualTo
-            }
-
-            test("Oppdatert sykmelding skal inneholde nye egenmeldingsdager") {
-                val sykmeldingId = UUID.randomUUID().toString()
-                val sendtSykmelding = getSendtSykmeldingKafkaMessage(sykmeldingId)
-
-                val egenmeldingsDager = listOf(LocalDate.of(2023, 1, 1))
-                val egenmeldingsSporsmål =
-                    SporsmalOgSvarDTO(
-                        "egenmeldingsdager",
-                        ShortNameDTO.EGENMELDINGSDAGER,
-                        SvartypeDTO.DAGER,
-                        objectMapper.writeValueAsString(egenmeldingsDager)
-                    )
-                sykmeldingService.handleSendtSykmeldingKafkaMessage(
-                    sykmeldingId,
-                    sendtSykmelding.copy(
-                        event = sendtSykmelding.event.copy(sporsmals = listOf(egenmeldingsSporsmål))
+                    val sykmelding = TestDb.getSykmelding(sykmeldingId)
+                    sykmelding?.egenmeldingsdager shouldBeEqualTo egenmeldingsDager
+
+                    val egenmeldingsDager2 = listOf(LocalDate.of(2023, 1, 5))
+
+                    val egenmeldingsSporsmål2 =
+                        SporsmalOgSvarDTO(
+                            "egenmeldingsdager",
+                            ShortNameDTO.EGENMELDINGSDAGER,
+                            SvartypeDTO.DAGER,
+                            objectMapper.writeValueAsString(egenmeldingsDager2),
+                        )
+                    sykmeldingService.handleSendtSykmeldingKafkaMessage(
+                        sykmeldingId,
+                        sendtSykmelding.copy(
+                            event =
+                                sendtSykmelding.event.copy(sporsmals = listOf(egenmeldingsSporsmål2)),
+                        ),
                     )
-                )
-                val sykmelding = TestDb.getSykmelding(sykmeldingId)
-                sykmelding?.egenmeldingsdager shouldBeEqualTo egenmeldingsDager
-
-                val egenmeldingsDager2 = listOf(LocalDate.of(2023, 1, 5))
 
-                val egenmeldingsSporsmål2 =
-                    SporsmalOgSvarDTO(
-                        "egenmeldingsdager",
-                        ShortNameDTO.EGENMELDINGSDAGER,
-                        SvartypeDTO.DAGER,
-                        objectMapper.writeValueAsString(egenmeldingsDager2)
-                    )
-                sykmeldingService.handleSendtSykmeldingKafkaMessage(
-                    sykmeldingId,
-                    sendtSykmelding.copy(
-                        event =
-                            sendtSykmelding.event.copy(sporsmals = listOf(egenmeldingsSporsmål2))
+                    val oppdatertSykmelding = TestDb.getSykmelding(sykmeldingId)
+                    oppdatertSykmelding?.egenmeldingsdager shouldBeEqualTo egenmeldingsDager2
+                }
+
+                test("Skal få true når det er active sykmeldinger") {
```

Comment:
mulig denne er litt forvirrende.
tittelen på denne og den under, sier at det er om man har aktive eller ikke sykmeldinger.
Men svaret vi får fra mock, inneholder ikke aktive siden man setter fom og tom til i fortiden.
Siden service bare sjekker om svaret er tomt for å gir true eller false.